src/rplidar_nodelet.cpp

```diff
@@ -280,6 +280,10 @@ namespace rplidar_ros {
       // But 99% of the time RPlidar produces 360 sized data
       if (node_count == 360) {  // Only publish when data size is 360
         pub->publish(scan_msg);
+      } else {
+        NODELET_WARN_STREAM("RPLidar : node_count is not equal to 360");
+        NODELET_WARN_STREAM("Node Count: " << node_count);
+        NODELET_WARN_STREAM("RPLidar: Not publishing data..");
```

Comment:
Can you squash these to one line for us, we have really alot of logging already.